### PR TITLE
Add script to automatically write a user to members.

### PR DESCRIPTION
**Pull request by tim-schilling**
This needs a few more things:
- Python management
- Docs
- Tests
- GitHub action to call this and create a PR

I don't know when I'll have time to pick this back up, so if anyone wants to take it over, please do!

### CODE REVIEW

**Comment by cunla** [comment]
Nicely done! I will know now who to reach out to with regex questions :) 

Not sure what you mean by python management here? This solution does not use any packages - are you thinking of future scripts or just for the GHA?

**Comment by tim-schilling** [comment]
Hahaha please no.

I was thinking we should be able to run and test this locally. I may be over thinking things again.

File: .github/workflows/add_member.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          git commit -m "Add ${{ inputs.username }} to django-commons"
+          git push origin add-user/${{ inputs.username }}
+
+      - name: Create pull request
+        run: |
+          gh pr create \
+            --title "Add ${{ inputs.username }} to django-commons" \
+            --body "Fix #${{ inputs.issue_number }}" \
+            --base main \
+            --head add-user/${{ inputs.username }}
```
Comment (tim-schilling) [inline .github/workflows/add_member.yml:56]: Oh, this is so much better than using an external action. Thank you!

Comment (cunla) [reply .github/workflows/add_member.yml:56]: Let’s see that it works :)
I haven't tried it before

**Comment by cunla** [comment]
@tim-schilling if you are ok with it, I can take over this. We can merge this PR and then I can run a few tries until I get it right :)

**Comment by tim-schilling** [comment]
Yeah, definitely!

**Pull request merged**